framework/src/main/java/io/github/kgress/scaffold/webdriver/WebDriverManager.java

```diff
@@ -211,7 +211,7 @@ private MutableCapabilities configureBrowserOptions() {
                                 .addArguments("--disable-extensions");
                     } else {
                         browserOptions = new ChromeOptions().setAcceptInsecureCerts(true)
-                        .addArguments("--window-size=" + screenResolution.getScreenShotResolutionAsString(SELENIUM));
+                                .addArguments("--window-size=" + screenResolution.getScreenShotResolutionAsString(SELENIUM));
```

Comment:
It looks like for the full sauce W3C compliance we will have to add the experimental option on chrome: `chOpts.setExperimentalOption("w3c", true);`

Reply:
oh, it looks like that is only for version 75 and lower. 

```For tests on Google Chrome version 74 or lower, the W3C capability must be set as an experimental option. ChromeDriver version 75 runs in W3C standard compliant mode by default, so setting this capability won't be necessary in the future.```

We should probably do a check on if the chrome version specified is 74 or lower, add that chrome option T.T

Reply:
does it break anything if it's set anyways for newer versions? it might be worth setting it and leaving it alone because parsing the version number feels like a more hacky solution:
```
var chromeOpts = new ChromeOptions()
	.setAcceptInsecureCerts(true)
	.addArguments("--window-size=" + screenResolution.getScreenShotResolutionAsString(SELENIUM));
if (desiredCapabilities.getBrowserVersion() != null) {
	try {
		if (Double.parseDouble(desiredCapabilities.getBrowserVersion().substring(0,2)) < 75) {
			chromeOpts.setExperimentalOption("w3c", true);
		}
	} catch (NumberFormatException e) {
		log.error("Unrecognized value passed for browserVersion, w3c compatibility flag not set");
	}
}
browserOptions = chromeOpts;
```

Reply:
Is it safe to assume browserVersion will always be an integer? I also noticed sauceLabs has "dev", "latest", "beta" as valid browserVersions available on their [platform configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/)

Reply:
ohhhh you're right. Version numbers can include letters and symbols.

How about we separate out that work into another ticket since it's more than just a quick fix.

Reply:
i just filed the ticket for the work: #67